### PR TITLE
Feat : drawer-screen improvements

### PR DESCRIPTION
**Pull request by surbhidavara99**

### CODE REVIEW

File: client/android/gradle.properties.example
[the remark this answers is not in the document]
Comment (surbhidavara99) [reply client/android/gradle.properties.example:1]: I have renamed it but it's still showing in the deleted format

File: client/screens/NavigationMobile.js
Comment (andrew-bierman) [inline client/screens/NavigationMobile.js:1]: Is signOut working on your end? Might need to add that into revised version

File: client/app/drawer/index.js
Comment (andrew-bierman) [inline client/app/drawer/index.js:1]: Should this be in component folder instead of app? Just for organization

Comment (surbhidavara99) [reply client/app/drawer/index.js:1]: We are making it full-screen so we need navigation to redirect it. That's why we can't move into a component folder

File: client/screens/NavigationMobile.js
Comment (surbhidavara99) [reply client/screens/NavigationMobile.js:1]: Yes, It's fixed. Please check now

**Pull request closed without merging**